dm/openapi/fixtures/source.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fixtures
+
+import (
+	"encoding/json"
+
+	"github.com/pingcap/ticdc/dm/openapi"
+)
+
+var sourceStr = `
+	{
+		"source_name": "mysql-01",
+		"host": "127.0.0.1",
+		"port": 3306,
+		"user": "root",
+		"password": "123456",
+		"enable_gtid": false,
+		"security": {
+		  "ssl_ca_content": "",
+		  "ssl_cert_content": "",
+		  "ssl_key_content": "",
+		  "cert_allowed_cn": [
+			"string"
+		  ]
+		},
+		"purge": {
+		  "interval": 3600,
+		  "expires": 0,
+		  "remain_space": 15
+		},
+		"relay_config": {
+		  "enable_relay": true,
+		  "relay_binlog_name": "mysql-bin.000002",
+		  "relay_binlog_gtid": "e9a1fc22-ec08-11e9-b2ac-0242ac110003:1-7849",
+		  "relay_dir": "./relay_log"
+		}
+	  }
+`
+
+// GenOpenAPISourceForTest generates openapi.Source for test.
```

Comment:
maybe this could be put in `xxx_test.go` file, test files are visible to each other and no need to bother normal code files.

Reply:
you mean just change the package name from `fixtures` to `openapi_test` ? if so i think there is no difference?

or move theso fixtures code to `openapi_test.go` and use `openapi` as package name?
i have try this way but failed, i can't call the func in `packageA_test.go` in `packageB_test.go`

Reply:
my bad, I didn't notice they are under different packages